Cargo.lock

```diff
@@ -49,6 +49,11 @@ dependencies = [
  "rustc_data_structures 0.0.0",
 ]
 
+[[package]]
```

Comment:
I would avoid including changes to `.lock` files in rollups :)

Reply:
Arg, didn't see it...